packages/dd-trace/test/plugins/agent.js

```diff
@@ -136,33 +149,11 @@ module.exports = {
     return promise
   },
 
-  // Return a promise that will resolve when all expectations have run.
-  promise () {
-    const promises = Array.from(handlers)
-      .map(handler => handler.promise.catch(e => e))
-
-    return Promise.all(promises)
-      .then(results => results.find(e => e instanceof Error))
-  },
-
   // Unregister any outstanding expectation callbacks.
   reset () {
     handlers.clear()
   },
 
-  // Wrap a callback so it will only be called when all expectations have run.
-  wrap (callback) {
-    return error => {
-      this.promise()
-        .then(err => callback(error || err))
-    }
-  },
-
-  // Return the current active span.
-  currentSpan () {
```

Comment:
This `currentSpan()` method removal might have been done accidentally. If the tests fail I'll revert this change.